FEXCore/Source/Interface/Config/Config.cpp

```diff
@@ -230,27 +230,26 @@ void Load() {
   }
 }
 
-fextl::string ExpandPath(const fextl::string& ContainerPrefix, fextl::string PathName) {
+fextl::string ExpandPath(const fextl::string& ContainerPrefix, const fextl::string& PathName) {
   if (PathName.empty()) {
     return {};
   }
 
-
   // Expand home if it exists
   if (FHU::Filesystem::IsRelative(PathName)) {
     fextl::string Home = getenv("HOME") ?: "";
     // Home expansion only works if it is the first character
     // This matches bash behaviour
     if (PathName.at(0) == '~') {
```

Comment:
Side note: Filenames can start with a tilde  😩 Should we check for `PathName.starts_with("~/")` instead hence?

Reply:
Err, nevermind. Since this is just Config code we can assume the filename is something reasonable.

Reply:
Doesn't really matter yea. if the user footgunned themselves with `~Config.json` or whatever then it's going to fail regardless.